examples/.env

```
@@ -19,6 +19,9 @@ REPLICATE_API_KEY=
 OLLAMA_HOST_URL=
 OLLAMA_MODEL=
 
+# For using Docker Model Runner
+DOCKER_MODEL_RUNNER_HOST_URL=
```

does it make sense to provide the default here as well?
```suggestion
DOCKER_MODEL_RUNNER_HOST_URL=http://127.0.0.1:12434
```

---

Port is customizable, I would prefer let it empty to "force" users to set it properly matching their settings in Docker Desktop.

---

I am in favor of adding it here, but add an output to the examples like "Calling on....."